bench-vortex/src/display.rs

```diff
@@ -81,12 +90,25 @@ pub fn print_measurements_json<T: ToJson>(all_measurements: Vec<T>) -> anyhow::R
     Ok(())
 }
 
-fn color(baseline_time: u128, test_time: u128) -> Color {
-    if test_time > (baseline_time + baseline_time / 2) {
-        Color::BG_RED
-    } else if test_time > (baseline_time + baseline_time / 10) {
-        Color::BG_YELLOW
-    } else {
-        Color::BG_BRIGHT_GREEN
+fn color(baseline: MeasurementValue, value: MeasurementValue, mode: RatioMode) -> Color {
+    match mode {
+        RatioMode::Time => {
+            if value > (baseline + baseline / 2) {
+                Color::BG_RED
+            } else if value > (baseline + baseline / 10) {
+                Color::BG_YELLOW
+            } else {
+                Color::BG_BRIGHT_GREEN
+            }
+        }
+        RatioMode::Throughput => {
+            if value < (baseline + baseline / 10) {
+                Color::BG_RED
+            } else if value < (baseline + baseline / 2) {
```

Comment:
If my throughput baseline is 100 MiB/s and I achieve 99 MiB/s, shouldn't I be green? Those are likely within the noise of one another.

I think it should be `if value < (baseline - baseline / 2)` etc. so 45 MiB/s is red, then 85 MiB/s is yellow, and 90 or greater MiB/s is green.